Makefile

```diff
@@ -134,6 +136,21 @@ ocs-registry-master:
 	@echo "Building ocs-registry image in appregistry format using master images for all OCS components"
 	hack/build-master-appregistry.sh
 
+catalog-build: opm
+	@echo "Building ocs-operator-catalog"
+	${OPM} render --output=yaml ${OCS_BUNDLE_IMG} > catalog/ocs-bundle.yaml
+	${OPM} render --output=yaml ${NOOBAA_BUNDLE_IMG} > catalog/noobaa-bundle.yaml
+	${OPM} validate catalog
+	${IMG_BUILD_CMD} build --platform ${TARGET_OS}/${TARGET_ARCH} -t ${OCS_CATALOG_IMG} -f Dockerfile.catalog .
+
+catalog-push:
+	@echo "Pushing ocs-operator-catalog image"
+	${IMG_BUILD_CMD} push ${OCS_CATALOG_IMG}
+
+opm: 
```

Comment:
Follow similar pattern like below and move this to end of file
https://github.com/red-hat-storage/ocs-operator/blob/2bfec67855b1a73972cbac052dfd12685f92186d/Makefile#L210-L221

Reply:
Instead of end as this is similar to ensuring operator-sdk moved it to below that. These all I will re arrange properly in a makefile refactor PR later.